android/app/build.gradle

```diff
@@ -145,7 +145,7 @@ repositories {
 
 configurations.all {
     resolutionStrategy {
-        force 'org.webkit:android-jsc:r236355'
+        force 'org.webkit:android-jsc:r241213'
```

Comment:
👋 @Tug , can you expand on the need to upgrade the JSC? Thanks!

Reply:
No need other than for the sake of being up to date.
I explored that first because I thought there was some issue with react-native but found out it wasn't the cause. 
In any case I can revert that if you don't think it's worth upgrading.

Reply:
> In any case I can revert that if you don't think it's worth upgrading.

Let's do that in a separate PR although I'd put low priority to that one anyway.

Should I assume that the other package version upgrades are also in a similar vein to upgrade in the sake of being up-to-date? If so, I'd also prefer those to be in separate PRs.

Long story short, if the only fix we need for the `hooks` is the deletion of the GB nested `node_modules` folder, let's only have that one in the PR. It makes things more clear, traceable and easier to merge.

Reply:
>I'd also prefer those to be in the separate PR.
👌